### PR TITLE
Add deprecated/non-standard badges

### PR DESCRIPTION
**Pull request by PositiveUser**
# Update MathML element reference page

## Description

Added some inline deprecated and non-standard annotations.

After opening some pages, I noticed that some elements were deprecated and/or non-standard, but it wasn't always mentioned in [this article](https://developer.mozilla.org/en-US/docs/Web/MathML/Element)

## Motivation

They help developers know deprecated and non-standard elements in [this article](https://developer.mozilla.org/en-US/docs/Web/MathML/Element) with a glance without opening the article of those elements.

### CODE REVIEW

File: files/en-us/web/mathml/element/index.md [hunk 1]
```diff
@@ -18,18 +18,18 @@ This is an alphabetical list of MathML elements. All of them implement the {{dom
...
 ### E
 
-- {{MathMLElement("menclose")}} (Enclosed contents)
+- {{MathMLElement("menclose")}} (Enclosed contents) {{non-standard_inline}}
 - {{MathMLElement("merror")}} (Enclosed syntax error messages)
 
 ### F
 
-- {{MathMLElement("mfenced")}} (Parentheses) {{deprecated_inline}}
+- {{MathMLElement("mfenced")}} (Parentheses) {{non-standard_inline}}{{deprecated_inline}}
```
Comment (Josh-Cena) [inline files/en-us/web/mathml/element/index.md:32]:
```suggestion
- {{MathMLElement("mfenced")}} (Parentheses) {{non-standard_inline}} {{deprecated_inline}}
```

**Review by Josh-Cena — COMMENTED** [review]
Could you also update the MathML elements by category list below?

**Comment by Josh-Cena** [comment]
@PositiveUser Once you update the list below we can merge

**Comment by PositiveUser** [comment]
It shows that one reviewer @fred-wang is pending

**Review by fred-wang — APPROVED** [review]
sorry for the delay. lgtm

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
@PositiveUser You don't necessarily have to wait for Frédéric's review; I can merge yours as well. However, we do need the two lists to be consistent.

**Review by PositiveUser — COMMENTED** [review]
What are the two lists? What do I need to do?
I never had a pull request experience like this before.

**Comment by PositiveUser** [comment]
Would it have been better if I opened a new, proper pull request?

**Comment by Josh-Cena** [comment]
On the page you edited, there are two lists: one called "[MathML elements A to Z](https://pr29204.content.dev.mdn.mozit.cloud/en-US/docs/Web/MathML/Element#mathml_elements_a_to_z)", one called "[MathML elements by category](https://pr29204.content.dev.mdn.mozit.cloud/en-US/docs/Web/MathML/Element#mathml_elements_by_category)". They contain exactly the same information, in different order. We need to keep the badges the same between them. It's okay if you want to open a new one, but you can also edit the existing one, by going to https://github.com/mdn/content/pull/29204/files and editing the file there.

**Comment by PositiveUser** [comment]
I opened a new pull request for this. Closing this.

**Pull request closed without merging**